src/solana/transaction.py

```diff
@@ -233,9 +236,9 @@ def compile_message(self) -> Message:  # pylint: disable=too-many-locals
 
         compiled_instructions: List[CompiledInstruction] = [
             CompiledInstruction(
-                accounts=[account_indices[str(am.pubkey)] for am in instruction.keys],
+                accounts=bytes([account_indices[str(am.pubkey)] for am in instruction.keys]),
                 program_id_index=account_indices[str(instruction.program_id)],
-                data=b58encode(instruction.data),
+                data=instruction.data,
```

Comment:
How come we don't have to encode base58 anymore?

Reply:
I actually don't know why we were doing it since the data was then being base58-decoded in `__encode_instruction` 🤷‍♂️ guessing this behaviour came from web3.js